bin/release-build.sh

```diff
@@ -12,5 +12,10 @@ if [[ $# != 1 ]]; then
 fi
 
 log_file="/build/logs/${1}.log"
-nohup time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir ${1%%-*} > $log_file 2>&1 &
+copy_dir="${1%%-*}"
+
+nohup sh -c \
+        "echo time ruby ${BUILD_DIR}/scripts/vmbuild.rb --type release --upload --reference $1 --copy-dir $copy_dir \
+         && time ruby ${BUILD_DIR}/scripts/vmpublish.rb --copy-dir $copy_dir" \
```

Comment:
1. Don't we have the same problem even with the current approach?  It seems each target has a hard-coded path to a .json file and we publish once per target. 

2. As you said, that's a valid point only if publish isn't a part of the release-build.sh!

3. I think this can go either way... It might make sense that we can publish some not all.  But since we're trying to publish 'release' builds only, if a release build failed for some, we might want to not publish any until we know what the problem is?